polkadot/runtime/rococo/src/impls.rs

```diff
@@ -29,14 +29,14 @@ use xcm_executor::traits::TransactAsset;
 /// remote calls.
 #[derive(Encode, Decode)]
 enum PeopleRuntimePallets<AccountId: Encode> {
-	#[codec(index = 50)]
-	Identity(IdentityCalls<AccountId>),
+	#[codec(index = 248)]
```

Comment:
looks like this references the index on the relay chain now.
may be TODO to correct this when we know the index on people chain

Reply:
Yeah, definitely